### PR TITLE
#134249289 redesign authentication ui

### PR DESCRIPTION
**Pull request by andela-oakinseye**
**What does this PR do?**
Redesign authentication pages adhering to the material design spec.

**Description of task to be completed**
This PR redesigns the login and logout pages to ensure a better user experience, it also fixes the issue with deployment to Heroku.

#### How should this be manually tested?
Visit the application home page and click on `login` or  `signup`.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#134249289

#### Screenshots (if appropriate)
screen shot 2016-12-07 at 11 36 04 am
screen shot 2016-12-07 at 11 35 32 am

#### Questions:
N/A

### CODE REVIEW

**Comment by raphaelakpan** [comment]
From the SignUp Screen shot above, you have a description above it saying "SIGN IN with" when you intended to sign up the user. You should correct that.

**Comment by andela-oakinseye** [comment]
@andela-rakpan 

Since we do not really put user's data in the DB, the social buttons would actually be signing in and not registering them.

File: public/js/mutateDOM.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/** Updates the DOM when nodes are added */
+/** State transitions cause node to be added  */
+/** References: https://getmdl.io/started/index.html , 
+ * https://developers.google.com/web/updates/2012/02/Detect-DOM-changes-with-Mutation-Observers
```
Comment (andela-rakpan) [inline public/js/mutateDOM.js]: Shorten line 8, or break into several lines

File: public/css/forms.css [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  width: 50px;
+}
+
+.avatar-list img{
+  height: 40px;
+  width: 40px; 
+}
+input{
+  outline: none;
+}
```
Comment (andela-aolaniran) [inline public/css/forms.css:54]: I thought we agreed on leaving a space before each opening brace

**Review by victor-enogwe — COMMENTED** [review]
looks good

**Comment by raphaelakpan** [comment]
Looks good.

**Review by neddinn — APPROVED** [review]

**Pull request merged**